esphome/components/toshiba/toshiba.cpp

```diff
@@ -490,7 +1038,155 @@ bool ToshibaClimate::is_valid_rac_pt1411hwru_message_(const uint8_t *message) {
   return false;
 }
 
+bool ToshibaClimate::process_ras_2819t_command_(const remote_base::ToshibaAcData &toshiba_data) {
+  // Check for power-off command (single packet)
+  if (toshiba_data.rc_code_2 == 0 && toshiba_data.rc_code_1 == RAS_2819T_POWER_OFF_COMMAND) {
+    ESP_LOGI(TAG, "RAS-2819T power OFF command detected!");
+    this->mode = climate::CLIMATE_MODE_OFF;
+    ESP_LOGI(TAG, "Mode: OFF");
+    this->publish_state();
+    return true;
+  }
+
+  // Check for swing toggle command (single packet)
+  if (toshiba_data.rc_code_2 == 0 && toshiba_data.rc_code_1 == RAS_2819T_SWING_TOGGLE) {
+    ESP_LOGI(TAG, "RAS-2819T swing toggle detected!");
+    // Toggle swing mode
+    if (this->swing_mode == climate::CLIMATE_SWING_VERTICAL) {
+      this->swing_mode = climate::CLIMATE_SWING_OFF;
+      ESP_LOGI(TAG, "Swing: OFF");
+    } else {
+      this->swing_mode = climate::CLIMATE_SWING_VERTICAL;
+      ESP_LOGI(TAG, "Swing: VERTICAL");
+    }
+    this->publish_state();
+    return true;
+  }
+
+  // Handle regular two-packet commands (mode/temperature/fan changes)
+  if (toshiba_data.rc_code_2 != 0) {
+    ESP_LOGI(TAG, "RAS-2819T protocol detected! rc_code_1=0x%" PRIX64 ", rc_code_2=0x%" PRIX64, toshiba_data.rc_code_1,
+             toshiba_data.rc_code_2);
+
+    // Convert to byte array for easier processing
+    uint8_t message1[6], message2[6];
+    for (int i = 0; i < 6; i++) {
+      message1[i] = (toshiba_data.rc_code_1 >> (40 - i * 8)) & 0xFF;
+      message2[i] = (toshiba_data.rc_code_2 >> (40 - i * 8)) & 0xFF;
+    }
+
+    ESP_LOGD(TAG, "[RX] Message1: %02X %02X %02X %02X %02X %02X", message1[0], message1[1], message1[2], message1[3],
+             message1[4], message1[5]);
+    ESP_LOGD(TAG, "[RX] Message2: %02X %02X %02X %02X %02X %02X", message2[0], message2[1], message2[2], message2[3],
+             message2[4], message2[5]);
```

Comment:
As above, these should probably be very_verbose:
```suggestion
    ESP_LOGVV(TAG, "[RX] Message1: %02X %02X %02X %02X %02X %02X", message1[0], message1[1], message1[2], message1[3],
             message1[4], message1[5]);
    ESP_LOGVV(TAG, "[RX] Message2: %02X %02X %02X %02X %02X %02X", message2[0], message2[1], message2[2], message2[3],
             message2[4], message2[5]);
```